ansible-tests/mistral/tripleo_validations/actions/list_validations.py

```diff
@@ -60,8 +64,8 @@ def get_remaining_metadata(validation):
 
 
 class ListValidations(base.Action):
-    def __init__(self):
-        pass
+    def __init__(self, groups=[]):
```

Comment:
Since we're not modifying `groups` anywhere in the code, this should be fine but in general it's safer to use `None` as the default argument value, because `[]` will be evaluated only once when Python loads the file and then reused every time `__init__` is called.

So this could bite us down the line.